pages/config.py

```diff
@@ -0,0 +1,54 @@
+import PySimpleGUI as sg
+import datetime
+import configparser
+
+current_year = datetime.datetime.now().year
+previous_year = current_year - 1
+
+layout = [
+    [sg.Text("Dados Pessoais")],
+    [sg.Text("CNPJ:"), sg.Input(size=(14, 1), key="-CNPJ-", enable_events=True)],
+    [sg.Text("Gerar DAS")],
+    [sg.Text("Mês:"), sg.Combo(['Janeiro', 'Fevereiro', 'Março', 'Abril', 'Maio', 'Junho', 'Julho', 'Agosto', 'Setembro', 'Outubro', 'Novembro', 'Dezembro'], key="-MONTH-", enable_events=True)],
+    [sg.Text("Ano:"), sg.Combo([str(current_year), str(previous_year)], default_value=str(current_year), size=(4), key="-YEAR-")],
+    [sg.Checkbox('Auto', default=False, key="-AUTO-", enable_events=True)],
+    [sg.Button("Salvar")],
+]
+
+window = sg.Window("Configurações", layout, size=(250, 215))
+
+config = configparser.ConfigParser()
+config.read('../env.txt')
+
+while True:
+    event, values = window.read()
+    if event == sg.WIN_CLOSED:
+        break
+    if event == "-CNPJ-":
+        print(values["-CNPJ-"])
+        if len(values["-CNPJ-"]) > 14:
+            values["-CNPJ-"] = values["-CNPJ-"][:14]
+            window["-CNPJ-"].update(value=values["-CNPJ-"])
+            sg.popup('A entrada deve ter exatamente 14 caracteres.')
+    if event == "-YEAR-":
+        print("-YEAR-")
+    if event == "-AUTO-":
+        if values["-AUTO-"]:
```

Comment:
o código pode ser reduzido, por exemplo:
Devido ao fato que se Auto for True, disabled vai ser true e se auto for false, disabled tmabém vai ser false.

IF auto
(espaço)   altera mes
(espaço)   altera ano
window["-MONTH-"].update(disabled=values["-AUTO-"])
window["-YEAR-"].update(disabled=values["-AUTO-"])